### ISSUE TITLE
version 5.0.5.0

### ISSUE THREAD

**Issue by SharadenduDwivedi** [yazdipour/unlimited-ticktick-windows#18]
**Describe the bug**
As soon as you click the settings button from profile, the app says "it encountered an error and needs to be closed"

![image](https://github.com/yazdipour/unlimited-ticktick-windows/assets/69635264/d0a24e23-f902-42c5-b4f5-467160ee77df)

**Comment by yazdipour** [comment]
I tried on my side. I dont see any errors! Is this happening all the times?

**Comment by SharadenduDwivedi** [comment]
yes sir, all the time (I am on windows 10)
so i reverted to the previous stable version for me, 5.0.4.0

**Comment by Viviore** [comment]
I have question, if I become a premium user on desktop by using this crack, does that means my apple phone gets premium as well since I'm using the same account?

**Comment by yazdipour** [comment]
@Viviore No. You are not becoming premium user! This tool just unlocks premium features on your PC.

**Comment by SharadenduDwivedi** [comment]
@Viviore I dont know about iOS but since I am on andriod, I downloaded the mod apk (cracked application package) and now that @yazdipour made a crack version for pc,

when I sign in into both the devices, it seems as if I have brought the premium.

See if you find something like that for iOS application package.

**Comment by SharadenduDwivedi** [comment]
@yazdipour
Greeting sir, I hope you're doing well!
If I could, can I make a request?
can something similar be done to todoist windows app?
just asking, feel free to turn down this request!

**Comment by sbatra-dev** [comment]
@SharadenduDwivedi 
No need to apologise :) 

I just came across TickTick and I can't really think of my life without it. Was using a never ending never sorted messenger chat for this 😭 

Huge tip about TickTick, keep it simple. Dump everything in inbox. At EOD, just assign priorities and deadlines, and maybe tags to separate work from personal. Nothing else or it becomes too much that you will wanna give up. I use it for habits as well. 

I downloaded the app from 4pda after spending hours after that message. From a trusted modder named (can't open page atm, it started with B)

Since I am using the habit tracker feature (love the widgets), using mods wont sync/save record of my habits beyond the 5 free ones (blocked by server and not the app so nothing nothing that mods can do). I have decided to buy anyways so much thanks for the reply. 

For anyone else reading, to download mods on 4pda, use mods posted by trusted members like nothingone, they aggregate mods from trusted members. They have mods for everything! 

It won't let you download the apks unless you make an account (worst part, it doesn't tell you that..). Use Google translate from Chrome to translate entire webpage. There is a very weird captcha at login, use Google lens on your mobile (none of the web apps using screenshot worked for me).

Have fun!

PSA: They are still some modded files from someone you don't know. Use at caution, i am using Fdroid + Shelter to be safer and different accounts, gmail and even in the apps. It would be even more trust worthy of they could have shared the instructions to mod rather than just the apk itself, something like what our guy @yazdipour did for us.
